WordPressUI/FancyAlert/FancyAlerts.storyboard

```diff
@@ -12,12 +12,12 @@
         <!--Fancy Alert View Controller-->
         <scene sceneID="C2X-eO-sRb">
             <objects>
-                <viewController id="ZA1-84-qnC" customClass="FancyAlertViewController" customModule="WordPressUI" customModuleProvider="target" sceneMemberID="viewController">
```

Comment:
This is needed because otherwise, Xcode will assume that `FancyAlertViewController` is in a module named `WordPressUIResources`, and crash.

Reply:
Was this done by hand, or within Interface Builder? Just thinking – we might need to do this elsewhere too? 🤔

Reply:
By hand, but it can be done in IB too by unchecking this box:

![59119126-e9d24400-8949-11e9-8db5-30e27dcf6b3e](https://user-images.githubusercontent.com/1773641/59130323-09c43080-8967-11e9-837d-9f428a9e079b.png)

This is the only storyboard in the pod so no other changes are needed. We should probably make similar changes to our other pods though.

I've opened a CocoaPods PR to remove the need for this here: https://github.com/CocoaPods/CocoaPods/pull/8890